freqtrade/edge/__init__.py

```diff
@@ -0,0 +1,388 @@
+# pragma pylint: disable=W0603
+""" Edge positioning package """
+import logging
+from typing import Any, Dict
+import arrow
+
+import numpy as np
+import utils_find_1st as utf1st
+from pandas import DataFrame
+
+import freqtrade.optimize as optimize
+from freqtrade.optimize.backtesting import BacktestResult
+from freqtrade.arguments import Arguments
+from freqtrade.exchange import Exchange
+from freqtrade.strategy.interface import SellType
+from freqtrade.strategy.resolver import IStrategy, StrategyResolver
+from freqtrade.optimize.backtesting import Backtesting
+
+
+logger = logging.getLogger(__name__)
+
+
+class Edge():
+
+    config: Dict = {}
+
+    def __init__(self, config: Dict[str, Any], exchange=None) -> None:
+        """
+        constructor
```

Comment:
instead of this docstring i would rather like docstrings on other functions (where appropriate) ...
everyone familiar with python will know this is a constructor ... (but it's a nitpick, really)